### PR TITLE
[VEGA-5444] layout changes

### PR DESCRIPTION
**Pull request by Alexandr1111**
## Problem statement
Произошли изменения дизайна в части анализа чувствительности и его статистики, таблицы рб-резалт в заголовках таблицы, а также в гистограммах.

https://artcpt.atlassian.net/browse/VEGA-5444
## Solution details

_{What has been done (list of details)}_
Задача Димы с макетами, согласно которым сделана текущая задача
https://artcpt.atlassian.net/browse/VEGA-5204
## Type

- [ ] Feature
- [ ] Bug Fix
- [x] Refactoring
- [ ] Test
- [x] Other

## Associated issues

_{Jira issues or PRs related to this one}_

## Quality control

- [x] PR: Based on a correct branch
- [x] PR: Head branch is rebased on the base branch
- [x] PR: Assignee chosen and necessary labels are set
- [x] PR: Current form is filled out (where it makes sense)
- [x] PR: Diff checked for irrelevant changes
- [x] PR: Commit messages follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specs
- [x] PR: Checked for spelling, grammar and typos
- [x] JS: There's no errors/warnings in the browser dev console
- [x] Layout: Tested with various content amounts
- [ ] Docs: All the important changes and features are described
- [ ] Tests: Configuration files are checked at test repositories
- [ ] Tests: New unit tests developed
- [ ] Tests: New E2E tests developed
- [x] Tests: Existing unit tests passed successfully
- [x] Tests: Existing E2E tests passed successfully
- [x] Tests: Manually tested on personal instance

## Notable statements
- [ ] Affects configuration files
- [ ] Brings new packages or updates existing
- [ ] Opened follow-up tasks to resolve

### CODE REVIEW

File: src/components/Histograms/chart/Chart.css [hunk 1]
```diff
@@ -63,14 +63,14 @@
   &__description {
     margin-bottom: var(--space-m);
 
-    font-size: 14px;
-    line-height: 140%;
+    font-size: 12px !important;
+    line-height: 150% !important;
```
Comment (SergeyKatugin) [inline src/components/Histograms/chart/Chart.css:67]: мб в Text вынести?

File: src/components/SensitiveAnalysis/chart/Chart.css [hunk 2]
```diff
@@ -96,8 +96,8 @@
   }
 
   &__text {
-    font-size: 14px;
-    line-height: 17px;
+    font-size: 12px;
+    line-height: 18px;
```
Comment (SergeyKatugin) [inline src/components/SensitiveAnalysis/chart/Chart.css:100]: в<Text> вынести бы

File: src/components/SensitiveAnalysis/statistic/SensitiveAnalysisStatistics.tsx [hunk 3]
```diff
@@ -52,14 +63,12 @@ const getMappedRow = (cells: SensitiveAnalysisStatisticCell[]): TableRow => {
...
-export const SensitiveAnalysisStatisticComponent: FC<Props> = ({
-  statistic,
-}) => {
+export const SensitiveAnalysisStatistics: FC<Props> = ({ statistic }) => {
   const columns: TableColumn<TableRow>[] = statistic?.headers.map(
     (header: SensitiveAnalysisStatisticHeaders) => {
       if (header?.children && header?.children?.length > 0) {
         return {
-          title: header.name,
+          title: header.name === 'НГЗ/НГР' ? 'НГЗ/НГР, млн.м³' : header.name,
```
Comment (SergeyKatugin) [inline src/components/SensitiveAnalysis/statistic/SensitiveAnalysisStatistics.tsx:71]: это из БЕ должно приходить, лучше убрать

Comment (Alexandr1111) [reply src/components/SensitiveAnalysis/statistic/SensitiveAnalysisStatistics.tsx:71]: Убираю

File: src/components/SensitiveAnalysis/chart/Chart.css (hunk 2 above)
Comment (Alexandr1111) [reply src/components/SensitiveAnalysis/chart/Chart.css:100]: Компонент косты внутри svg, принадлежащей d3? Я бы не городил и в данном случае оставил бы как было

Comment (SergeyKatugin) [reply src/components/SensitiveAnalysis/chart/Chart.css:100]: а, понял, можно коммент дописать, для таких как я)

**Review by ndrzhn — APPROVED** [review]

**Pull request merged**